[DOCS] [Maps] Add documentation for custom icons (#129530)
elastic/kibana: docs/maps/vector-style-properties.asciidoc

[role="screenshot"]
image::maps/images/maki-icons.png[]

Custom Icons

You may also use your own SVG icon to style Point features in your map. In **Layer settings** click the **Add custom icon** button from the *Icon* dropdown list to upload your icon to the map. For best results, your SVG icon should be monochrome and have limited details. Dynamic styling in **Elastic Maps** requires rendering SVG icons as PNGs using a https://en.wikipedia.org/wiki/Signed_distance_function[Signed Distance Function]. As a result, sharp corners and intricate details may not render correctly. Modifying the settings under Advanced Options may improve rendering.

> Custom icons can be added, edited, or deleted in the **Map settings** for your map. 

Reviewer: ```suggestion
To add, edit, or delete custom icons, use the **Map settings** for your map. 
```

> Custom icons can be added, edited, or deleted in the **Map settings** for your map. 

Reviewer: ```suggestion
To add, edit, or delete custom icons, use the **Map settings** for your map. 
```
```suggestion
Custom icons can be added, edited, or deleted in the **Map settings** for your map. 
```

[role="screenshot"]
image::maps/images/maki-icons.png[]

Custom Icons

You can also use your own SVG icon to style Point features in your map. In **Layer settings** open the *icon* dropdown, and click the **Add custom icon** button. For best results, your SVG icon should be monochrome and have limited details. 

> Dynamic styling in **Elastic Maps** requires rendering SVG icons as PNGs using a https://en.wikipedia.org/wiki/Signed_distance_function[Signed distance function]. As a result, sharp corners and intricate details may not render correctly. Modifying the settings under **Advanced Options** in the **Add custom icon** modal may improve rendering.

Reviewer: ```suggestion
Dynamic styling in **Elastic Maps** requires rendering SVG icons as PNGs using a https://en.wikipedia.org/wiki/Signed_distance_function[signed distance function]. As a result, sharp corners and intricate details may not render correctly. Modifying the settings under **Advanced Options** in the **Add custom icon** modal may improve rendering.
```

> To add, edit, or delete custom icons use the **Map settings** for your map. 

Reviewer 2: How about changing to "Manage custom icons in <<maps-settings, Map settings>>". This will also add a link to map settings section
